modules/virt-creating-offline-vm-snapshot-cli.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-offline-vm-snapshots.adoc
+
+[id="virt-creating-offline-vm-snapshot-cli_{context}"]
+= Creating an offline virtual machine snapshot in the CLI
+
+You can create a virtual machine (VM) snapshot for an offline VM by creating a `VirtualMachineSnapshot` object.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Power down the VM for which you want to create a snapshot.
+
```

Comment:
In order for disk content to be saved, the PVCs must be in a storage class with CSI snapshots configured.

Reply:
Added this requirement to prerequisites